pkg/resourcemapping/operator_resource_mapping.go

```diff
@@ -266,20 +280,37 @@ func (ormClient *ORMClient) LocateOwnerPaths(ownedObj *unstructured.Unstructured
 	ownedKey := ownedObj.GetKind() + "/" + ownedObj.GetName() // source or owned resource
 
 	var allOwnerResourcePaths map[string][]devopsv1alpha1.ResourcePath
+	owners := map[string][]devopsv1alpha1.ResourcePath{}
 
-	orm, ownerObj, err := ormClient.retrieveORM(ownedObj, ownerReference)
+	orm, ownerObj, err := ormClient.retrieveOwnerResource(ownedObj, ownerReference)
 	if err != nil {
 		return allOwnerResourcePaths, err
 	}
 	if orm == nil {
-		return allOwnerResourcePaths, fmt.Errorf("Cannot find orm V1 for source: %s\n", ownedKey)
-	}
-	if orm == nil || ownerObj == nil {
-		return allOwnerResourcePaths, fmt.Errorf("Cannot find owner for source: %s\n", ownedKey)
+		return allOwnerResourcePaths, fmt.Errorf("cannot find orm V1 for source: %s", ownedKey)
+	}
+	// if owner resource obj not found then return source/owned obj in the owner reosurce paths
+	// to be consistent with ORM V2 flow
+	if ownerObj == nil {
+		var ownedRef v1.ObjectReference = v1.ObjectReference{
```

Comment:
maybe this could be extracted into a small helper function for better readability, since this is duplicated in multiple places.

Reply:
Will be addressed